test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/Dockerfile

```diff
@@ -9,7 +9,7 @@ WORKDIR /w3c
 RUN apt-get update && apt-get install -y git
 RUN git clone https://github.com/w3c/trace-context.git
 
-FROM mcr.microsoft.com/dotnet/core/sdk:${SDK_VERSION} AS build
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build
```

Comment:
@eddynaka Not sure I solved this in the best way. I hardcoded 3.1 SDK for the build stage and then left the final stage using the `SDK_VERSION` arg. Can you think of a better way to deal with this?

Reply:
That's the way. Since 2.1 only exists for adk 3.*, we will have to use that to build. The important part is the version below. So we are still testing for both.